package.json

```diff
@@ -1,20 +1,19 @@
 {
-  "name": "visual",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "d3-array": "^1.2.1",
-    "d3-fetch": "^1.1.0",
-    "d3-scale": "^2.0.0",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2",
-    "react-materialize": "^2.2.1",
-    "react-scripts": "1.1.4"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
-  }
+    "scripts": {
+        "client": "cd client && yarn start",
```

Comment:
add the following to avoid the need to manually npm install both here and in the client
```
"postinstall": "cd client && npm install",
```

Reply:
(make sure to update your readme if you make this change)

Reply:
This `npm install` is only for client-side node modules, right?
What about the server-side node modules?
Should I change the scripts to:
```
"postinstall": "cd client && npm install && cd .. && npm install",
```
?

Reply:
no, `postinstall` happens after you run `npm install`.  The user just runs `npm install` as normal and it will do both.